setup.py

```diff
@@ -1,5 +1,31 @@
 from setuptools import setup, find_packages
 
+
+def union(*deps):
+    return list(set().union(*[extras[dep] for dep in deps]))
+
+extras = {
+    "envs": [
+        "atari_py~=0.2.0",            # atari environments
+        "box2d-py~=2.3.5",            # box2d environments
```

Comment:
Wouldn't it be better to do "gym[atari,box2d]>=0.17.2" in principle? This way Gym would specify versions of the packages it wants. As a real example of this, the ALE changes us, OpenAI and MILA/DeepMind montreal have been working on are going to result in dramatic changes to gym[atari] in the next ~6 months.

Reply:
It's going to move from atari-py to ale-py, amongst other things

Reply:
See above. I agree in principle but it doesn't work unless we remove `gym` from the core dependencies.